src/provider/camunda/properties/IdProps.js

```diff
@@ -50,7 +50,11 @@ function Id(props) {
     return getBusinessObject(element).get('id');
   };
 
-  const setValue = (value) => {
+  const setValue = (value, error) => {
+    if (error) {
```

Comment:
I assume there was a test for this behavior already and that test is still working?

Reply:
This behavior is tested here: https://github.com/bpmn-io/dmn-js-properties-panel/blob/87001449325955ce2487cce3ca2a271679943e1d/test/spec/provider/dmn/IdProps.spec.js#L106-L233